modules/validations-before-agent-iso-creation.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * installing/installing-with-agent/installing-with-agent.adoc
+
+:_content-type: REFERENCE
+[id="validations-before-agent-iso-creation_{context}"]
+= Validation checks before agent.iso creation
+
+The Agent-based Installer performs validation checks on user defined yaml files before the ISO is created. Once the validations are successfull, the `agent.iso`
```

Comment:
YAML (at least, that's how we used it in the guidelines: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference)